exampleCourse/questions/demoAutograderSquare/tests/initial_code.py

```diff
@@ -0,0 +1,5 @@
+import numpy as np
+
+# This default code is defined in "initial_code.py"!
```

Comment:
```suggestion
# This default code is defined in "initial_code.py"
```

Doesn't seem worthy of an exclamation point 😄

Reply:
🙃